cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetricsTable.java

```diff
@@ -41,6 +41,11 @@
   void put(SortedMap<byte[], ? extends SortedMap<byte[], Long>> updates);
 
   /**
+   * Write multiple rows, each with multiple individual columns of byte array type to write.
+   */
+  void putBytes(SortedMap<byte[], ? extends SortedMap<byte[], byte[]>> updates);
```

Comment:
This method is quite confusing.

1. It is not exactly part of `MetricsTable`, but we have it here for laziness, since the transport offset is stored in a separate table.
2. Having `byte[]` as keys and values with proper documentation making this method very unclear.